### PR TITLE
Dashboard Header style changes

### PR DESCRIPTION
**Pull request by ayeswarya**
Resolves #

## Description
Changes to Dashboard header styles

## Proposed Changes

-

## Screenshots

Screen Shot 2021-03-04 at 3 27 08 PM

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: packages/dashboards/addon/components/dashboard-filters.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+interface Args {
+  dashboard: DashboardModel;
+  onRemoveFilter(filter: FilterFragment): void;
+  onUpdateFilter(filter: FilterFragment): void;
+  onAddFilter(): void;
+}
+
+export default class DashboardFiltersComponent extends Component<Args> {
+  @service
+  store!: Store;
```
Comment (kevinhinterlong) [inline packages/dashboards/addon/components/dashboard-filters.ts:22]:
```suggestion
  declare store: Store;
```

File: packages/reports/addon/components/favorite-item.hbs [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+<div
+  role="button"
+  class="favorite-item {{if this.isFavorite "favorite-item--active"}}"
+  {{on "click" @onToggle}}
+  ...attributes
+>
+  <DenaliIcon
+    @size="small"
+    @icon={{if this.isFavorite "star-solid" "star"}}
+    ...attributes
```
Comment (kevinhinterlong) [inline packages/reports/addon/components/favorite-item.hbs:11]: Double `...attributes`

File: packages/reports/addon/components/favorite-item.hbs [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{{!-- Copyright 2020, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
```
Comment (kevinhinterlong) [inline packages/reports/addon/components/favorite-item.hbs:1]:
```suggestion
{{!-- Copyright 2021, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
```

File: packages/reports/addon/components/favorite-item.hbs (hunk 2 above)
Comment (jkusa) [reply packages/reports/addon/components/favorite-item.hbs:11]:
```suggestion
```

File: packages/dashboards/addon/components/dashboard-filters.ts [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+ * Copyright 2021, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { computed } from '@ember/object';
+import { inject as service } from '@ember/service';
+import DashboardModel from 'navi-core/models/dashboard';
+import FilterFragment from 'navi-core/models/bard-request-v2/fragments/filter';
+import Store from '@ember-data/store';
```
Comment (kevinhinterlong) [inline packages/dashboards/addon/components/dashboard-filters.ts:11]:
```suggestion
import type DashboardModel from 'navi-core/models/dashboard';
import type FilterFragment from 'navi-core/models/bard-request-v2/fragments/filter';
import type Store from '@ember-data/store';
```

**Review by kevinhinterlong — APPROVED** [review]

File: packages/reports/addon/components/favorite-item.ts [hunk 5]
```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2021, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+import Component from '@glimmer/component';
+import DeliverableItemModel from 'navi-core/models/deliverable-item';
+import UserModel from 'navi-core/models/user';
```
Comment (kevinhinterlong) [inline packages/reports/addon/components/favorite-item.ts:7]:
```suggestion
import type DeliverableItemModel from 'navi-core/models/deliverable-item';
import type UserModel from 'navi-core/models/user';
```

**Pull request merged**